nautobot/extras/tests/test_views.py

```diff
@@ -437,6 +438,35 @@ def test_view_object_with_custom_link(self):
         self.assertIn(f"FOO {site.name} BAR", content, content)
 
 
+class DynamicGroupTestCase(
+    ViewTestCases.CreateObjectViewTestCase,
+    ViewTestCases.DeleteObjectViewTestCase,
+    ViewTestCases.EditObjectViewTestCase,
+    ViewTestCases.GetObjectViewTestCase,
+    ViewTestCases.GetObjectChangelogViewTestCase,
+    ViewTestCases.ListObjectsViewTestCase,
+    ViewTestCases.BulkDeleteObjectsViewTestCase,
```

Comment:
This isn't using `ViewTestCases.PrimaryObjectViewTestCase` because we don't yet have bulk-import or bulk-edit views for DynamicGroup yet? Is that worth adding a TODO comment for?